tests/api/v1/generators/put.js

```diff
@@ -69,7 +69,7 @@ describe('tests/api/v1/generators/put.js >', () => {
           description: 'An ok sample\'s value, e.g. \'0\'',
         },
       },
-      subjects: ['US'],
+      subjectQuery: '?subjects',
```

Comment:
`?subjects` doesn't seem like a valid subjectQuery... shouldn't it be something like `?absolutePath=Foo.*` or `?tags=Bar`

Reply:
I was just using the fields that were already defined in utils.getGenerator. I will change it.